app/core/reservationbook.py

```diff
@@ -2,53 +2,90 @@
 from waiting import Waiting
 from collections import deque
 from datetime import datetime
+from datetime import timedelta
 from app.mapper import ReservationMapper
 from app.mapper import WaitingMapper
+from app.mapper import TimeslotMapper
+
+
 # ReservationBook object
 class ReservationBook:
-
     # Constructor
-    def __init__(self,reservationlist,waitinglist):
+    def __init__(self, reservationlist, waitinglist):
         self.reservationList = reservationlist
         self.waitingList = waitinglist
 
     # Method to make a reservation
     def makeReservation(self, room, holder, time, description):
+
         # Check if room is available at specifie time
         if (self.available(room, time) == True):
-            r = Reservation(room,holder,time,description)
+            r = Reservation(room, holder, time, description)
             ReservationMapper.registerNew(r)
             ReservationMapper.done()
             self.reservationList.append(r)
 
+    def makeRepeatedReservation(self, room, user, timeslot, description, repeat_amount):
+        max_repetition = 2
```

Comment:
I prefer keeping 0 repetition because if you don't want any repetition then it should be 0.